DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/EmailDelegates/EmailDelegatesViewModelTests.cs

```diff
@@ -0,0 +1,211 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.EmailDelegates
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.EmailDelegates;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class EmailDelegatesViewModelTests
+    {
+        private readonly DelegateUserCard[] delegateUsers =
+        {
+            new DelegateUserCard
+                { Id = 1, FirstName = "a", LastName = "Surname", Answer4 = string.Empty },
+            new DelegateUserCard
+                { Id = 2, FirstName = "b purple", LastName = "Surname", Answer4 = string.Empty },
+            new DelegateUserCard
+                { Id = 3, FirstName = "c", LastName = "Surname" },
+            new DelegateUserCard
+                { Id = 4, FirstName = "d purple", LastName = "Surname" },
+            new DelegateUserCard
+                { Id = 5, FirstName = "e", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 6, FirstName = "f", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 7, FirstName = "g", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 8, FirstName = "h", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 9, FirstName = "i", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 10, FirstName = "j", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 11, FirstName = "k", LastName = "Surname", DateRegistered = DateTime.Today },
+            new DelegateUserCard
+                { Id = 12, FirstName = "l", LastName = "Surname", Answer4 = "C 1" },
+            new DelegateUserCard
+                { Id = 13, FirstName = "m", LastName = "Surname", Answer4 = "C 2" },
+            new DelegateUserCard
+                { Id = 14, FirstName = "n", LastName = "Surname", Answer4 = "C 2" },
+            new DelegateUserCard
+                { Id = 15, FirstName = "o", LastName = "Surname", DateRegistered = DateTime.Today.AddDays(1) }
+        };
+
+        [Test]
+        public void EmailDelegatesViewModel_should_return_all_delegates_on_one_page()
+        {
+            // When
+            var model = new EmailDelegatesViewModel(
+                delegateUsers,
+                new List<(int, string)>(),
+                new List<CustomPrompt>(),
+                null
+            );
+
+            // Then
+            model.Delegates!.Should().HaveCount(delegateUsers.Length);

```

Comment:
Should Delegates be optional on the model? 🤔

Reply:
When submitting the form, Delegates will not be set so we can't assume it's not null.